src/keycloak/chart/templates/uds-package.yaml

```diff
@@ -59,6 +59,17 @@ spec:
           app.kubernetes.io/name: keycloak
         port: {{ .Values.postgresql.port }}
         remoteGenerated: Anywhere
+      # HA for keycloak
+      - direction: Ingress
+        remoteGenerated: IntraNamespace
+        ports:
+          - 7800
+          - 57800
+      - direction: Egress
+        remoteGenerated: IntraNamespace
+        ports:
+          - 7800
+          - 57800
```

Comment:
nit: this could/should be wrapped in a conditional on `autoscaling.enabled`

Reply:
Updated